test_utils/tasks.py

```diff
@@ -0,0 +1,26 @@
+"""
+Tasks used in tests
```

Comment:
Hmm.. aren't these specific to `PersistOnFailureTask`?  Why are they pulled out into a generic test-utils?